packages/web3-common/src/errors.ts

```diff
@@ -398,14 +408,28 @@ export class AbiError extends Web3Error {
 
 export class PrivateKeyLengthError extends Web3Error {
 	public code = ERR_PRIVATE_KEY_LENGTH;
-	public constructor(value: string | Buffer) {
-		super(`Invalid value given "${value.toString()}". Error: Private key must be 32 bytes.`);
+	public constructor() {
+		super(`Error: Private key must be 32 bytes.`);
 	}
 }
 
 export class InvalidPrivateKeyError extends Web3Error {
 	public code = ERR_INVALID_PRIVATE_KEY;
-	public constructor(value: string | Buffer) {
-		super(`Invalid value given "${String(value)}". Error: not a valid string or buffer.`);
+	public constructor() {
+		super(`Error: not a valid string or buffer.`);
+	}
+}
+
+export class SignerError extends Web3Error {
+	public code = ERR_SIGNATURE_FAILED;
+	public constructor(errorDetails: string) {
+		super(`Error: Invalid signature. "${errorDetails}"`);
```

Comment:
```suggestion
		super(`Invalid signature. "${errorDetails}"`);
```

No need to have prefix `Error: ` JavaScript's native Error class will append it to the message